### ISSUE TITLE
[Bug Report][3.3.21] No screen reader accessible announcement for "No data available" in v-autocomplete

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#18459]
### Environment
**Vuetify Version:** 3.3.21
**Vue Version:** 3.3.4
**Browsers:** Chrome 117.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
With a screen reader on (like VoiceOver for Mac) go to the autocomplete in the example and type something that is not a value that can be selected (i.e. 'test'). The label 'No Data Available' will show, but the screen reader does not announce in any way that there are no values to choose from.

### Expected Behavior
There should be a div with class sr-only that announces the change, or some similar screen reader accessible announcement such as with an aria-live region

### Actual Behavior
There is no screen reader announcement
<img width="1508" alt="Screenshot 2023-10-13 at 8 16 15 AM" src="https://github.com/vuetifyjs/vuetify/assets/60188830/b04f42cb-7f21-4460-aba8-fe2deba7697e">

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVT8sOgjAQ/JVmL1wU4uNk0MSQ6A+YeAAOFVbSpHSbUojG+O+WIiKnnZ2d3Z1JX9CYIjpqHXYtwg5ii7WW3OIhU4zF3ZJr7aFvClKWC4XmSw2K1lJBbgstjjRjkt9Q7jM4/k9hmu+E+9Q4QRokXIo7GSV4sGBBQpIML6nHJwdF6ekzkqkGxQUfvOnB9Um1UFWQT4cPcTR3NJp39Ny9F/pwcfQLDe8FbMJNuIWhrleQfwDa2lpB)
